src/saucelabs/runner.js

```diff
@@ -95,7 +95,7 @@ export default class SauceLabsRunner extends EventEmitter {
     async _checkForFreeMachines () {
         var freeMachinesCount = await this._getFreeMachineCount();
 
-        if (freeMachinesCount)
+        if (freeMachinesCount > 0)
```

Comment:
I think method `_getFreeMachineCount` schould return `Math.max(0, ...)` to code will be more clearly